### PR TITLE
📨 feat: implemented user invite with sendgrid

### PR DESCRIPTION
**Pull request by Lyton505**
This PR sends the user an email invite with send grid using the domain test505@lyton.dev. I also removed node_modules from tracking

image

### CODE REVIEW

File: api/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.env.local
+**/dist/
+node_modules/
```
Comment (Lyton505) [inline api/.gitignore:3]: @lolitaroz should I remove this and have you add it as a fix on main before I rebase? Including node_modules probably makes my pr harder to review

**Comment by Lyton505** [comment]
new design

image

File: api/.gitignore (hunk 1 above)
Comment (lolitaroz) [reply api/.gitignore:3]: yes that would be great, please don't commit node_modules lol!

Comment (lolitaroz) [reply api/.gitignore:3]: i added overall git ignore pushes so that should be available too, making yaml now

**Comment by Lyton505** [comment]
@lolitaroz let me know if this pr needs any more changes. If you want to test it out, the secrets needed are in [Notion](https://www.notion.so/changepp/Secrets-139aeb87757080cd9822d86e4e1b2588?pvs=4https://www.notion.so/changepp/Secrets-139aeb87757080cd9822d86e4e1b2588?pvs=4)

**Review by lolitaroz — APPROVED** [review]
great job! merge it!

**Pull request merged**